decisions/0015-env-triad-terms.md

```diff
@@ -1,52 +1,124 @@
+extracting subsets from those ontologies will be queries composed with
+the [Ontology Access Kit](https://github.com/INCATools/ontology-access-kit). For example, NMDC may set the general rule
+that the values for the `env_broad_scale`, in combination with all environmental sample types, must be subclasses
+of [biome [ENVO:00000428]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00000428?lang=en).
+In that case, the `env_broad_scale` values sets for individual environments/sample types must be subsets of the general
+query.
+
+The empirical evidence is based on values that have been used for the environmental triad slots in prominent Biosample
+metadata systems, such as NCBI, GOLD, and NMDC. Some of those sources, especially NCBI, are very permissive for the
+environmental triad values. Because the candidate value tables include multiple empirical sources plus the results of
+the general, rules-based OAK queries, no one source can introduce inappropriate values nor exclude reasonable values.
+Initial exploration suggests that the queries will be hard limits for `env_broad_scale` and `env_medium`, but that the
+empirical evidence will be more important for `env_local_scale`. General, rules-based queries will be refined for each
+of the environments (MIxS Extensions), and will be reflected in either query-specific columns or general boolean
+columns, like 'is_biome' or 'is_environmental_material'.
+
+- The environmental triad squad is developing an NMDC-prioritized list of environments. Soil will be completed first.
+- This ADR will be updated as environment-specific queries are created.
+- The logic described above is intended to minimize cherry-picking of values for the sets, but individual squad members
+  will be given the opportunity to add their own `Y` or `N` votes for each value in the tables. It may be possible to
+  feed the insights from these votes back into reusable logic, but that might require complex OAK queries over
+  inconsistent annotations, or grouping of values within a semantic embedding space. For example, there may be evidence
+  that [farm [ENVO:00000078]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00000078),
+  [agricultural field [ENVO:00000114]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00000114)
+  and [banana plantation [ENVO:00000161]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00000161)
+  should all be included in the `env_local_scale` enumeration for Soil samples. It may be more expeditious for
+  a human to make the judgment call to include farm and exclude the others, compared to the effort required to automate
+  that. Visualizations will be provided to illustrate these cases. See the end of this ADR.
```

Comment:
```suggestion
  Expert review will be done on an initial list of accepted terms. Insights from review will be fed back into reusable logic, which that might require complex OAK queries. over
  inconsistent annotations, or grouping of values within a semantic embedding space. For example, there may be evidence
  that [farm [ENVO:00000078]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00000078),
  [agricultural field [ENVO:00000114]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00000114)
  and [banana plantation [ENVO:00000161]](https://www.ebi.ac.uk/ols4/ontologies/envo/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FENVO_00000161)
```

Too much for a short ADR. we can add complex text to the final version if it's necessary.